### PR TITLE
AWS- SQS added credentials - access and secret keys

### PR DESCRIPTION
**Pull request by tallieber**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33752

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/AWS-SQS/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### AWS - SQS
+- Added support of *access key* and *secret key* on instance configuration 
+- Upgraded the Docker image to demisto/boto3:2.0.0.19307
```
Comment (bakatzir) [inline Packs/AWS-SQS/ReleaseNotes/1_1_0.md:5]:
```suggestion
- Upgraded the Docker image to: *demisto/boto3:2.0.0.19307*.
```

File: Packs/AWS-SQS/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### AWS - SQS
+- Added support of *access key* and *secret key* on instance configuration
```
Comment (bakatzir) [inline Packs/AWS-SQS/ReleaseNotes/1_1_0.md:4]:
```suggestion
- Added support of *access key* and *secret key* to the instance configuration.
```

File: Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.yml [hunk 3]
```diff
@@ -48,6 +44,28 @@ configuration:
...
   required: false
   type: 13
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
```
Comment (bakatzir) [inline Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.yml:54]: proxy and trust should be at the bottom. `isFetch` up top.

File: Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py [hunk 4]
```diff
@@ -33,30 +45,69 @@ def aws_session(service='sqs', region=None, roleArn=None, roleSessionName=None,
     elif AWS_rolePolicy is not None:
         kwargs.update({'Policy': AWS_rolePolicy})
 
-    if kwargs:
-        sts_client = boto3.client('sts')
+    if kwargs and not AWS_ACCESS_KEY_ID:
```
Comment (bakatzir) [inline Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py:48]: can you inline comment for the different auth cases? i'm not sure I can understand where the old case is where a user does not configure `AWS_ACCESS_KEY_ID` nor `AWS_SECRET_ACCESS_KEY`.

**Review by bakatzir — COMMENTED** [review]
Also - are we not adding an instance config to content-test-conf?

File: .circleci/config.yml [hunk 5]
```diff
@@ -1071,7 +1071,6 @@ jobs:
             ./Tests/scripts/slack_notifier.sh 'test_playbooks' ./env_results.json
           when: always
       - *comment_on_contrib_pr
-      - *destroy_instances
```
Comment (hod-alpert) [inline .circleci/config.yml:1074]: ?

**Review by hod-alpert — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 5 above)
Comment (bakatzir) [reply .circleci/config.yml:1074]: just for testing, she wants to log into the instance

**Comment by bakatzir** [comment]
@tallieber for the content branch to run with your changes in content-test-conf, branch names in both repos needs to be the same.

**Review by hod-alpert — APPROVED** [review]

File: Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py [hunk 6]
```diff
@@ -33,30 +45,69 @@ def aws_session(service='sqs', region=None, roleArn=None, roleSessionName=None,
...
+        client = boto3.client(
+            service_name=service,
+            region_name=AWS_DEFAULT_REGION,
+            aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
+            aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
+            aws_session_token=sts_response['Credentials']['SessionToken'],
+            verify=VERIFY_CERTIFICATE,
+            config=config
+        )
+    else:  # login with access key id, anf if is None than permissions pulled from the EC2 metadata
```
Comment (bakatzir) [inline Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py:94]:
```suggestion
    else:  # login with access key id, and if region is None than permissions pulled from the EC2 metadata
```

**Review by bakatzir — COMMENTED** [review]
please re generate the readme and mention the 3 different options of signing in.

**Review by bakatzir — APPROVED** [review]

**Pull request merged**